src/components/table.js

```diff
@@ -58,6 +58,50 @@ const activityColumns = (object)=>[
             }
         }
     },
+    ...object=='repo'?
+        [{
+            title: t('大屏'),
+            dataIndex: 'name',
+            align:'center',
+            width: '20%',
+            //根据行号判断，前300的项目可点击大屏
+            render: function (text, row, index) {
+                //console.log('andy',row);
+                let [org_name,repo_name]=text.split("/");
+                //console.log('year',year);
+                //console.log('month',month);
```

Comment:
Hi, @andyhuang18, I have some suggestions :)

1. It's better to remove useless code such as `console.log` before creating a PR